Porting/perldocstyle.pod

```diff
@@ -0,0 +1,1087 @@
+the nonetheless-technical topics at hand.
+
+Like the best written works, Perl's documentation has a soul. Get
+familiar with it as a reader to internalize its voice, and then find
+your own way to express it in your own contributions. Writing clearly,
+succinctly, and with knowledge of your audience's expectations will get
+you most of the way there, in the meantime.
+
+Every line in the docs--whether English sentence or Perl statement --
+should serve the purpose of bringing understanding to the reader. Should
+a sentence exist mainly to make a wry joke that doesn't further the
+reader's knowledge of Perl, set it aside, and consider recasting it into
+a personal blog post or other article instead.
+
+Write with a light heart, and a miserly hand. 
+
+=head1 INDEX OF PREFERRED TERMS
+
+L<As noted above|/Choice of underlying style guide: CMOS>, this guide
+"inherits" all the preferred terms listed in the Chicago Manual of
+Style, 17th edition, and adds the following terms of particular interest
+to Perl documentation.
+
+=over
+
+=item *
+
+B<built-in function>
+
+Not "builtin".
+
+=item *
+
+B<Darwin>
+
+See L<macOS|"macOS">.
+
+=item *
+
+B<macOS>
+
+Use this term for Apple's operating system instead of "Mac OS X" or
+variants thereof.
+
+This term is also preferable to "Darwin", unless one needs to refer
+to macOS's Unix layer specifically.
+
+=item *
+
+B<man page>
+
+One unit of Unix-style documentation. Not "manpage". Preferable to "manual page".
+
+=item *
+
+B<Perl; perl>
+
+The name of the programming language is Perl, with a leading capital
+"P", and the remainder in lowercase. (Never "PERL".)
+
+The interpreter program that reads and executes Perl code is named
+"C<perl>", in lowercase and in monospace (as with any other command
+name).
+
+Generally, unless you are specifically writing about the
+command-line C<perl> progam (as, for example, L<C<perlrun>|perlrun>
+does), use "Perl" instead.
+
+=item *
+
+B<Perl 5>
+
+Documentation need not follow Perl's name with a "5", or any other
+number, except during discussions of Perl's history, future plans,
+or explicit comparisons between major Perl versions.
+
+Before 2019, specifying "Perl 5" was sometimes needed to distinguish
+the language from Perl 6. With the latter's renaming to "Raku", this
+practice became unnecessary.
+
+=item *
+
+B<Perl 6>
+
+See L<Raku|"Raku">.
+
+=item *
+
+B<Perl 5 Porters, the; porters, the; p5p>
+
+The full name of the team responsible for Perl's ongoing maintenance
+and development is "the Perl 5 Porters", and this sobriquet should
+be spelled out in the first mention within any one document. It may
+thereafter call the team "the porters" or "p5p".
+
+Not "Perl5 Porters".
+
+=item *
+
+B<program>
+
+The most general descriptor for a stand-alone work made out of
+executable Perl code. Synonymous with, and preferable to, "script".
+
+=item *
+
+B<Raku>
+
+Perl's "sister language", and the current open-source language
```

Comment:
"current" is too time-sensitive for this document. Perhaps:

Perl's "sister language", developed in a project initially known as "Perl 6" led by Larry Wall, Perl's original designer.